src/main/java/com/minecolonies/coremod/colony/buildings/BuildingFarmer.java

```diff
@@ -2,7 +2,8 @@
 
 import com.minecolonies.api.util.InventoryUtils;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused import 'com.minecolonies.api.util.InventoryUtils'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AUselessImportCheck)